cea/demand/demand_main.py

```diff
@@ -69,60 +71,72 @@ def demand_calculation(locator, weather_path, gv, use_dynamic_infiltration_calcu
     t0 = time.clock()
 
     # weather model
-    weather_data = epwreader.epw_reader(weather_path)[['drybulb_C', 'wetbulb_C', 'relhum_percent', 'windspd_ms', 'skytemp_C']]
+    weather_data = epwreader.epw_reader(weather_path)[
+        ['drybulb_C', 'wetbulb_C', 'relhum_percent', 'windspd_ms', 'skytemp_C']]
 
     building_properties, schedules_dict, date = properties_and_schedule(gv, locator)
 
     # in case gv passes a list of specific buildings to simulate.
-    if gv.simulate_building_list:
-        list_building_names = gv.simulate_building_list
-    else:
+    if buildings == []:
         list_building_names = building_properties.list_building_names()
+        print('Running demand calculation for all buildings in the zone')
+    else:
+        list_building_names = gv.simulate_building_list
```

Comment:
didn't you just add a `config.demand.buildings` variable to use instead of `gv.simulate_building_list`? (or did I misunderstand?)

Reply:
my bad. I corrected it